src/Spark/version.i

```diff
@@ -1 +1 @@
-&GLOBAL-DEFINE SPARK_VERSION 6.0.0-2020.12.03.074908 (11.7.7)
\ No newline at end of file
+&GLOBAL-DEFINE SPARK_VERSION 6.0.2-2022.02.22.073132 (12.5.0)
```

Comment:
what does the OE version here mean? This preprocessor is going to be the same for all versions. right? even when running 11.7 or 12.2.

I see that this is overwritten by the build script. It may be useful to say so in a comment in this include, and/or remove the text in here

Reply:
This is just used for compilation purposes and is not used afterwards. Yes, it changes and gets checked in but is irrelevant for operation.

Reply:
The file should now be removed after compilation and not checked in